geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/RedisListTest.java

```diff
@@ -143,6 +143,28 @@ public void versionDoesNotUpdateWhenReferenceElementNotFound() {
     assertThat(list.getVersion()).isEqualTo(originalVersion);
   }
 
+  @Test
+  public void versionDoesNotUpdateWhenLtrimDoesNotModifyList() {
+    Region<RedisKey, RedisData> region = uncheckedCast(mock(PartitionedRegion.class));
+    RedisList list = createRedisListWithDuplicateElements();
+
+    byte originalVersion = list.getVersion();
+    list.ltrim(0, 5, region, null);
```

Comment:
To make it more obvious what this is doing, could the end index be `-1`? That way, regardless of the size of the list, we're guaranteed to always do a no-op.

Reply:
Done.